README.md

```
@@ -108,18 +108,43 @@ git clone https://github.com/cyberagiinc/DevDocs.git
+
+# Ensure NEXT_PUBLIC_BACKEND_URL in .env is set correctly (e.g., http://localhost:24125)
+# This allows the frontend (running in your browser) to communicate with the backend service.
+
+
 # Start all services using Docker
 ./docker-start.sh
 ```
 
-For Windows users:
+For Windows users: Experimental Only (Not Tested Yet)
 ```cmd
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)